llama_cpp/llama.py

```diff
@@ -243,7 +242,7 @@ def __init__(
             )  # keep a reference to the array so it is not gc'd
             self.model_params.tensor_split = self._c_tensor_split
         self.model_params.vocab_only = vocab_only
-        self.model_params.use_mmap = use_mmap if lora_path is None else False
+        self.model_params.use_mmap = use_mmap
```

Comment:
Memory mapping is supported for LoRAs now because llama.cpp no longer merges the LoRA into the base model, so the original model GGUF is unchanged and can be mapped.

See the equivalent change in llama.cpp:
https://github.com/ggerganov/llama.cpp/pull/8332/files#diff-201cbc8fd17750764ed4a0862232e81503550c201995e16dc2e2766754eaa57aL688